docs/guides/slp-software-setup.md

```diff
@@ -0,0 +1,367 @@
+}
+```
+
+After this, we will have generated the three necessary bits of information we need:
+
+**The WIF (private key)**    
+`L1aeBNDeQ43PtFN2Jc9tbeqhNiqBgDHbxMctefHftHw4WtSo621e`
+
+**The recieving SLP address, make sure to add the prefix ‘simpleledger:’**    
+`simpleledger:qzvjfs5tuw56est4mwtzw3fegucgdwr00v0nkx3qwh`
+
+**And the funding cashaddr format address:**    
+`bitcoincash:qzvjfs5tuw56est4mwtzw3fegucgdwr00vrgaayqsf`
+
+
+> **TESTNET**    
+> For the testnet version, just add the flag `--testnet` in each query
+
+
+
+### Option 2: bitcore
+
+Bitcore is a library used by SLPJS which, aside from many other things, allows us to create and manage new addresses using many different methods, like seed based, WIF based, or without any arguments at all.
+
+We can generate our new addresses using the following script. If you are not sure how to run it, use the same method as the balance.js and genesis.js described in the next section of the guide.
+
+### bitcoreAddr
+
+```javascript
+const bitcore = require('bitcore-lib-cash');
+
+var privateKey = new bitcore.PrivateKey();
+
+var publicKey = new bitcore.PublicKey(privateKey)
+
+console.log("WIF:", privateKey.toWIF());
+
+console.log("cashaddr:", publicKey.toAddress().toString());
+```
+
+Result
+
+```bash
+> npm run-script bitcoreAddr
+
+> <folder>@1.0.0 bitcoreAddr C:\Users\<user>\<folder>
+> node bitcoreAddr.js
+
+WIF: Ky3QwBS8gKD2Cdu4iUe5aaqojsU8BfuT4Gf8zMT11AdsEkcN9UJj
+cashaddr: bitcoincash:qp8x5ppmu4d0h2g2mgu405jkp5r4x87h7yya64tfd6
+```
+
+This will give us:
+
+**The WIF (private key)**    
+`Ky3QwBS8gKD2Cdu4iUe5aaqojsU8BfuT4Gf8zMT11AdsEkcN9UJj`
+
+**And the funding cashaddr format address:**    
+`bitcoincash:qp8x5ppmu4d0h2g2mgu405jkp5r4x87h7yya64tfd6`
+
+Now to obtain the **recieving SLP address** we can use multiple different methods, like calling the Electron Cash CLI described in the previous section, or using any other resource that performs the conversion, like for example the [bitcoin.com explorer](https://explorer.bitcoin.com/bch) which given a cashaddr format will display the corresponding simpleledger address. In this case:
+
+**Recieving SLP address**    
+`simpleledger:qp8x5ppmu4d0h2g2mgu405jkp5r4x87h7ygx3w7fny`
+
+> **TESTNET**    
+> For the testnet version, just add the following as the second line in the `bitcoreAddr.js` file    
+> ```javascript
+> bitcore.Networks.defaultNetwork = bitcore.Networks.testnet;
+> ````
+
+
+## Scripts
+
+Now that everything is set up, and we have all the information we need, we can take care of the code.
+
+Note that for these examples, we will be using the addresses we created using the Electron Cash SLP method.
+
+After running init, you should have a `package.json` file created in the same folder you have run the command. In this file you can find a section called `scripts`. There you can define various scripts you will be able to easily run with commands as follows
+
+```bash
+> npm run-script <script-name>
```

Comment:
you can do `npm run <script-name>`  which is shorter

Reply:
Huh, I think I tried while doing the setup and didn't work. Probably nothing worked back then. Good to know.
Fixed all instances.